packages/connect-conformance/package.json

```diff
@@ -3,21 +3,22 @@
   "version": "1.4.0",
   "private": true,
   "type": "module",
-  "main": "./dist/cjs/index.js",
+  "main": "./dist/cjs/src/index.js",
   "exports": {
     ".": {
-      "import": "./dist/esm/index.js",
-      "require": "./dist/cjs/index.js"
+      "import": "./dist/esm/src/index.js",
+      "require": "./dist/cjs/src/index.js"
     }
   },
   "bin": {
-    "connectconformance": "bin/connectconformance"
+    "connectconformance": "bin/connectconformance.cjs"
   },
   "scripts": {
     "generate": "buf generate buf.build/connectrpc/conformance:v1.0.2",
     "clean": "rm -rf ./dist/*",
     "build": "npm run build:cjs && npm run build:esm",
-    "build:cjs": "tsc --project tsconfig.json --module commonjs --verbatimModuleSyntax false --moduleResolution node10 --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
+    "postbuild": "connectconformance --version",
```

Comment:
npm runs "postbuild" right after "build". This downloads the conformance runner. Running `connectconformance` in parallel won't try to download multiple times.